imap_processing/ultra/l1b/ultra_l1b_extended.py

```diff
@@ -35,8 +34,8 @@ class StopType(Enum):
 
     Top = 1
     Bottom = 2
-    PH: ClassVar[list[int]] = [1, 2]
-    SSD: ClassVar[list[int]] = [8, 9, 10, 11, 12, 13, 14, 15]
+    PH = [1, 2]  # noqa RUF012 mutable class attribute
```

Comment:
@laspsandoval mypy complained if this was annotated, and ruff complains if this isn't annotated :) No satisfying our overlords here so I ignored this for now.